plugin-management-cli/src/main/java/io/jenkins/tools/pluginmanager/cli/CliOptions.java

```diff
@@ -106,6 +106,11 @@
     @Option(name = "--help", aliases = {"-h"}, help = true)
     private boolean showHelp;
 
+    @Option(name = "--skip-failed-plugins", usage = "Set to true to skip installing plugins that have failed to download. " +
```

Comment:
won't you end up with a broken jenkins installation if this happens though?

Reply:
Based off of what I'm reading from the codebase, if this is an existing jenkins instance being upgraded, then the plugin in question being upgraded will simply be left in place at the current version no?

Reply:
hmm yeah I guess

Reply:
Can you think of anything off the top of your head that would help indicate that this isn't a fresh install and is an upgrade? Then I could add a clause that doesn't allow the flag to be usable on an instance with no plugins as I could definitely see that being an issue.

Reply:
I would probably just document it as a dangerous option, and it might cause failures.

Reply:
Added it to the document @timja :)